src/Http/MiddlewareChain.php

```diff
@@ -0,0 +1,79 @@
+<?php
+namespace App\Http;
+
+use Psr\Http\Server\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * MiddlewareChain is a chaining of middlewares that implements MiddlewareInterface
+ */
+class MiddlewareChain implements MiddlewareInterface
```

Comment:
This one should go into yiisoft/yii-web.

Reply:
I'll handle adding it.

Reply:
https://github.com/yiisoft/yii-web/commit/727a39171c395616cf694092403a01e608d1ed24